test/integration/start_stop_delete_test.go

```diff
@@ -153,6 +153,14 @@ func TestStartStop(t *testing.T) {
 					if err != nil {
 						t.Errorf("%s failed: %v", rr.Args, err)
 					}
+
+					rr, err = Run(t, exec.CommandContext(ctx, "kubectl", "config", "get-contexts"))
```

Comment:
Instead of pattern matching, can I suggest using:

`kubectl config get-contexts <profile>`

Then you can check for an error code. For example:

```
if rr.ExitCode != 1 {
  t.Errorf("wanted exit code 1, got %d. output: %s", rr.ExitCode, rr.Output())
}
```

Reply:
DONE